Changelog.md

```diff
@@ -10,6 +10,7 @@ Compiler Features:
 Bugfixes:
  * Assembler: Prevent incorrect calculation of tag sizes.
  * SMTChecker: Fix internal error caused by not respecting the sign of an integer type when constructing zero-value SMT expressions.
+ * SMTChecker: Ensure query is properly flushed to a file before calling solver when using SMT-LIB interface.
```

Comment:
Would be best to add the changelog entries already in the proper sorting order.

Reply:
Sorry, was in a bit of a hurry! Will keep it in mind!